curriculum/challenges/english/03-front-end-libraries/react/review-using-props-with-stateless-functional-components.english.md

```diff
@@ -23,15 +23,15 @@ The code editor has a <code>CampSite</code> component that renders a <code>Campe
 ```yml
 tests:
   - text: The <code>CampSite</code> component should render.
-    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('CampSite').length === 1; })(), 'The <code>CampSite</code> component should render.');
+    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('CampSite').length === 1; })());
   - text: The <code>Camper</code> component should render.
-    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('Camper').length === 1; })(), 'The <code>Camper</code> component should render.');
+    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('Camper').length === 1; })());
   - text: The <code>Camper</code> component should include default props which assign the string <code>CamperBot</code> to the key <code>name</code>.
-    testString: getUserInput => assert((function() { const noWhiteSpace = getUserInput('index').replace(/\s/g, ''); const verify1 = 'Camper.defaultProps={name:\'CamperBot\'}'; const verify2 = 'Camper.defaultProps={name:"CamperBot"}'; return (noWhiteSpace.includes(verify1) || noWhiteSpace.includes(verify2)); })(), 'The <code>Camper</code> component should include default props which assign the string <code>CamperBot</code> to the key <code>name</code>.');
+    testString: assert(/Camper.defaultProps={name:['"]CamperBot['"],?}/.test(code.replace(/\s/g, '')));
```

Comment:
Might I suggest the following to allow single backticks and simplify the regex more?
```suggestion
    testString: assert(/Camper.defaultProps={name:(['"`])CamperBot\1,?}/.test(code.replace(/\s/g, '')));
```

Reply:
Sure, makes sense. I think the editor's js parser will flag up syntactically invalid code, but it can't hurt to have a better regex.